docs/configuration/secrets/extract-codacy-secrets.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+
+usage()
+{
+    echo "Usage: $0 -n <NAMESPACE>"
+    echo "Example: '$0 -n codacy' would retrieve secrets from the logs from the 'codacy' namespace of the kubernetes cluster"
```

Comment:
I'm thinking we could have an extra line just to say what this script does. And then the example could become a bit simpler:

```suggestion
    echo "Retrieves secrets used to encrypt sensitive data on the database from the Codacy logs.
    echo "Usage: $0 -n <NAMESPACE>"
    echo "Example: '$0 -n codacy' retrieves secrets from Codacy running on the cluster namespace 'codacy'"
```

Does this make sense?